packages/react/src/Label/Label.tsx

```diff
@@ -8,13 +8,27 @@ import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { ForwardedRef, LabelHTMLAttributes, PropsWithChildren } from 'react'
 
+export type LabelProps = PropsWithChildren<LabelHTMLAttributes<HTMLLabelElement>> & {
+  /** Text to inform the user that the field is explicitely not required. */
+  notRequiredLabel?: string
+  /** Whether the field set is required. */
+  required?: boolean
+}
+
+export const LabelOptionalFlag = ({ notRequiredLabel }: LabelProps) => (
+  <span className="ams-label__not-required">
+    <span className="ams-visually-hidden">,</span> ({notRequiredLabel})
+  </span>
+)
+
 export const Label = forwardRef(
   (
-    { children, className, ...restProps }: PropsWithChildren<LabelHTMLAttributes<HTMLLabelElement>>,
+    { children, className, notRequiredLabel = 'niet verplicht', required, ...restProps }: LabelProps,
```

Comment:
I'm not sure we should hard-code the `notRequired` part like this. [NLDS suggests either marking all fields that are not required, or all fields that are retuired, with a preference for the first one](https://www.nldesignsystem.nl/richtlijnen/formulieren/voorkom-fouten/verplichte-velden/#vermeld-duidelijk-of-een-veld-wel-of-niet-verplicht-is). If we hard-code it as not required, we lose the second option. 

Maybe we can make it more generic by using a `suffix` or `tag` prop or something, which can be used for both options. What do you think?

Reply:
We decided to use the concept of "hint".